addons/account_edi_ubl_cii/models/res_partner.py

```diff
@@ -75,6 +75,8 @@ class ResPartner(models.Model):
             ('0213', "0213 - Finnish Organization Value Add Tax Identifier"),
             ('0215', "0215 - Net service ID"),
             ('0216', "0216 - OVTcode"),
+            ('0221', "0221 - The registered number of the qualified invoice issuer (Japan)"),
+            ('0230', "0230 - National e-Invoicing Framework (Malaysia)"),
```

Comment:
You need to rebase since those values are already deployed on 17.0:
https://github.com/odoo/odoo/commit/67b52999f261e795ed28d8cb972cbd7718724303